src/agentplatform/drift/checker.py

```
@@ -92,6 +92,10 @@ def check_workspace(
         top = rel.split("/", 1)[0]
         if top in RUNTIME_OWNED:
             continue
+        # Python 字节码缓存：live 执行 import 渲染脚本的运行时副产物，
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)